defaults/main.yml

```diff
@@ -704,6 +854,11 @@ owncloud__role_recommended_config:
   ## ISO 8601 datetime: 2004-02-12T15:19:21+00:00
   logdateformat: 'Y-m-d H:i:s.u'
 
+  ## Default is "stable" as of Nextcloud 11.
+  ## "production" is the most conservative channel which is preferred by the role maintainers.
+  ## https://owncloud.org/release-channels/
+  'updater.release.channel': 'production'
```

Comment:
`updater.release.channel` comes from Nextcloud. I have not seen this setting in ownCloud 9 yet. ownCloud should just ignore it when it does not understand it.